<!DOCTYPE html>
<html lang="en">
    <head>
        <meta charset="utf-8"/>
        <meta name="description" content="Visualization and integration of TCGA data.">
        <meta property="og:title" content="MEXPRESS"/>
        <meta property="og:url" content="http://mexpress.be"/>
        <meta property="og:site_name" content="MEXPRESS"/>
        <meta property="og:type" content="website"/>
        <meta property="og:image" content="http://mexpress.be/images/plot.png"/>
        <meta property="og:description" content="Visualization and integration of TCGA data."/>
        <title>mexpress</title>
        <link rel="icon" type="image/png" href="images/m_icon.png">
        <link rel="stylesheet" type="text/css" href="css/global.css">
        <link rel="stylesheet" type="text/css" href="css/about.css">
        <link href='http://fonts.googleapis.com/css?family=Dosis:800|Open+Sans:600' rel='stylesheet' type='text/css'>
        <script>
            (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
            (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
            m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
            })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
            ga('create', 'UA-54184541-2', 'auto');
            ga('send', 'pageview');
        </script>
        <!--[if lt IE 9]>
        <script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script>
        <![endif]-->
    </head>
    <body>
        <div class="container">
            <header>
                <h1 class="mainTitle">m<span class="letterE">e</span>x<span class="letterP">p</span><span class="letterR">r</span>ess</h1>
                <p class="subTitle">
                    a methylation &amp; expression visualization tool
                </p>
                <a href="/"><img class="logo" src="images/m.png"/></a>
            </header>
            <div class="content">
                <section>
                    <h2>What?</h2>
                    <p>
                        <span class="mexpress">MEXPRESS</span> is a data visualization tool designed for the easy visualization of <a href="https://tcga-data.nci.nih.gov/tcga/" target="_blank" title="This link will take you to the TCGA website.">TCGA</a> expression, DNA methylation and clinical data, as well as the relationships between them.
                        You can find all the details in our <a href="http://www.biomedcentral.com/1471-2164/16/636" target="_blank">publication</a>.
                    </p>
                </section>
                <section>
                    <h2>How?</h2>
                    <p>
                        Easy! Just pick a gene and cancer type you are interested in and MEXPRESS will show you the corresponding expression, DNA methylation and clinical data, as well as the relationship between these data sets.
                    </p>
                    <h3>Getting started</h3>
                    <p>
                        This is the MEXPRESS start screen:
                    </p>
                </section>
                <figure>
                    <img alt="This the MEXPRESS start screen." src="images/mexpressStartScreen.png"/>
                </figure>
                <section>
                    <h3>Selecting data</h3>
                    <p>
                        On the left side you will notice the text input field where you can enter a gene name (both HGNC symbols and Ensembl gene IDs are recognized) as well as the list of cancer types you can choose from.
                    </p>
                    <p>
                        Let's say you are interested in the expression and methylation data of the GSTP1 gene in prostate cancer.
                        Simply enter GSTP1 in the text field and select prostate adenocarcinoma (PRAD).
                    </p>
                </section>
                <figure>
                    <img alt="MEXPRESS is now ready to plot." src="images/mexpressReadyToPlot.png"/>
                </figure>
                <section>
                    <h3>Creating the plot</h3>
                    <p>
                        All that is left to do after selecting a gene and cancer type is clicking the <span class="blue">plot</span> button and the data you selected will be visualised.
                        Samples are arranged from left to right, while the different data types (clinical, expression and methylation) are arranged from the top to the bottom of the plot.
                    </p>
                    <p>
                        Hovering over one of the methylation line plots will bring up the ID of the corresponding probe.
                        Click on a methylation line plot to fix the probe ID on the figure.
                        Click it again to remove the probe ID (or click another methylation line plot).
                        You can also highlight the promoter probes by clicking the button right above the legend.
                    </p>
                    <p>
                        If you would like to download the figure you just created, simply click the <span class="blue">png</span> or <span class="blue">svg</span> button in the upper right corner (depending on which image format you want).
                    </p>
                </section>
                <figure>
                    <img alt="This is the final MEXPRESS plot." src="images/mexpressPlot.png"/>
                </figure>
                <section>
                    <p>
                        If you would like to emphasize the probes that are located in a gene's promoter region, you click the <span class="grey">highlight promoter probes</span> button.
                        This will turn the highlighting of the promoter probes on as you can see in the image below.
                    </p>
                </section>
                <figure>
                    <img alt="Highlighting the promoter probes." src="images/mexpressPromoterHighlighted.png"/>
                </figure>
                <section>
                    <h3>About the plot</h3>
                    <p>
                        At the top of the figure you find the HGNC gene symbol, the Ensembl gene ID and the genomic location of GSTP1.
                        The gene symbol link will open the <a href="http://www.genecards.org" target="_blank">GeneCards</a> page about GSTP1 in a new tab or window.
                    </p>
                    <p>
                        Next up is the legend of the plot, which adapts to the cancer type you selected, and below the legend is the actual plot.
                        From top to bottom you will see the clinical data, the expression values and the methylation data.
                        Let's take a detailed look at each part.
                    </p>
                    <h4>expression data</h4>
                    <figure>
                        <img alt="A closer look at the expression data." src="images/mexpressExpression.png"/>
                    </figure>
                    <p>
                        The height of the orange line represents the logarithm of the level 3 RNA-sequencing data in TCGA (normalized RNASeqV2 values per gene).
                        The expression data forms the basis of the whole plot, because the samples are ranked based on their expression value for the gene we selected with the highest expression on the left side and the lowest on the right.
                    </p>
                    <h4>methylation data</h4>
                    <figure>
                        <img alt="A closer look at the methylation data." src="images/mexpressMethylation.png"/>
                    </figure>
                    <p>
                        On the left hand side, you see the GSTP1 gene, two CpG islands (in green) and the different GSTP1 transcripts.
                        The arrow on the gene indicates its direction.
                        When&#8211;like in this example&#8211;the arrow points down, the gene is located on the &#43; strand. If it points up, the gene lies on the &#45; strand.
                    </p>
                    <p>
                        On the right hand side, you see all the Infinium 450k probes that are linked to GSTP1.
                        The height of the blue lines indicates the beta value for a probe. When there is no data available for a certain probe, no line is plotted and instead it simply says "no data".
                        Gaps in the line indicate that there was no methylation data for one or more samples.
                    </p>
                    <p>
                        Just as for the expression data, the samples are ranked along the x axis (they were ordered based on their GSTP1 expression value).
                        Thin blue lines connect the probes to their respective genomic locations.
                        Hovering your mouse cursor over a methylation data plot will highlight the corresponding probe on the left hand side and the name of the probe will be shown.
                        You can fix the highlighting of a probe by clicking on its data plot.
                        Clicking the same data plot a second time will clear the highlighting.
                    </p>
                    <figure>
                        <img alt="Highlighting a methylation probe." src="images/mexpressProbeHighlighted.png"/>
                    </figure>
                    <p>
                        Once you have fixed a probe's highlighting by clicking on the data plot, you can click on the probe's name to reveal the probe's genomic location and annotation.
                    </p>
                    <figure>
                        <img alt="Revealing a probe's annotation." src="images/mexpressProbeAnnotation.png"/>
                    </figure>
                    <p>
                        The values on the far right represent the Pearson product&#45;moment correlation coefficient between the methylation values for a probe and the expression values.
                        For GSTP1 you can see that there are quite a few probes with a strong negative correlation between methylation and expression, indicating that GSTP1 expression might be controlled through DNA methylation (which has already been described, see <a href="http://www.nature.com/onc/journal/v18/n6/full/1202415a.html" target="_blank">Millar <em>et al.</em>, 1999</a>).
                        As the plot's legend explains, the asterisks give you an indication of the significance of the correlations.
                    </p>
                    <h4>clinical data</h4>
                    <figure>
                        <img alt="A closer look at the clinical data." src="images/mexpressClinical.png"/>
                    </figure>
                    <p>
                        For every cancer type, the most relevant clinical parameters have been extracted from TCGA.
                        In order to represent all the data as barplots, some clinical parameters have been converted to numeric values.
                        One example is the pathologic stage (not shown in this example) where values such as <span class="value">Stage IIA</span> and <span class="value">Stage IV</span> were converted to the values <span class="value">2</span> and <span class="value">4</span> respectively.
                    </p>
                    <p>
                        The names of the different clinical parameters are listed on the left and the <a href="http://en.wikipedia.org/wiki/Pearson_correlation_coefficient" target="_blank">Pearson product-moment correlation values</a> or the p values for <a href="http://en.wikipedia.org/wiki/Mann%E2%80%93Whitney_U_test" target="_blank">Wilcoxon rank-sum test</a> can be found on the right.
                        Whenever a clinical parameter contains only two levels (e.g. male or female) a p value is calculated instead of a correlation coefficient.
                        This p value indicates the difference in expression between the two groups for this parameter.
                        For the sample type parameter, the expression is always compared between the normal and tumor samples.
                    </p>
                    <p>
                        The sample type annotation can be found in the legend of the plot.
                        In this case the dark blue samples are normal whereas the light blue bars indicate tumor samples.
                        Again, the findings from <a href="http://www.nature.com/onc/journal/v18/n6/full/1202415a.html" target="_blank">Millar <em>et al.</em> (1999)</a> are confirmed, because it is clear that the normal samples tend to have a higher GSTP1 expression than the tumor samples.
                    </p>
                    <p>
                        As described earlier, the samples are sorted based on their expression value by default.
                        By clicking on the name of the annotation parameter you're interested in, you can rearrange the samples by the annotation you just selected.
                        So if you would for example like to compare age to the expression and methylation of a certain gene or to the other clinical parameters, you just have to click on "age at diagnosis" and the samples will be reordered.
                    </p>
                </section>
                <section>
                    <a class="startButton" href="/">Let's get started!</a>
                </section>
            </div>
            <footer>
                <p>
                    <em>By downloading, analyzing, and/or utilizing TCGA data for publication purposes, the user accepts the data use restrictions and requirements as outlined in the TCGA Publication Guidelines.
                    See <a href="http://cancergenome.nih.gov/abouttcga/policies/publicationguidelines" target="_blank">http://cancergenome.nih.gov/abouttcga/policies/publicationguidelines</a> for additional information.</em>
                </p>
                <p>
                    If you would like to use a MEXPRESS plot in your publication (and we hope you do!), please cite our paper <a href="http://www.biomedcentral.com/1471-2164/16/636" target="_blank">MEXPRESS: visualizing expression, DNA methylation and clinical TCGA data</a>.
                </p>
                <p>
                    Developed by Alexander Koch &#64; <a href="http://www.biobix.be" target="_blank">BIOBIX</a> &#8211; Ghent University
                </p>
                <p>
                    Contact: <a href="mailto:alexander.koch@ugent.be?Subject=MEXPRESS">alexander.koch@ugent.be</a>
                </p>
                <p>
                    <a href="http://mexpress.be">MEXPRESS</a> &#8211; 2016
                </p>
            </footer>
        </div>
    </body>
</html>    